pkg/config/images.go

```diff
@@ -1,12 +1,29 @@
 package config
 
 import (
+	"fmt"
 	"os"
+	"regexp"
 	"strings"
 
 	acsapi "github.com/openshift/openshift-azure/pkg/api"
 )
 
+func imageVersion(incVersion string) string {
+	// Version should be in xxx.xx.xxxxxxxx format
+	// e.g: 310.14.20180101
+	rxVersion := regexp.MustCompile(`^([0-9]){1}([0-9]{1,3})` +
```

Comment:
use strings.Split

Reply:
I looked into string.split but wasn't aware of a clean way to divide the numbers exactly how we wanted them.
`310.14.20180101` to `3.10.14`

If you have an example that'd be great.